packages/react-meteor-data/ReactMeteorData.jsx

```diff
@@ -180,6 +180,9 @@ export default function connect(options) {
       getMeteorData() {
         return getMeteorData(this.props);
       }
+			getWrappedComponent() {
```

Comment:
Would you mind fixing the spacing here to align with the style used in the rest of the file?